src/Block.js

```diff
@@ -1,67 +1,65 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { View, StyleSheet, SafeAreaView } from 'react-native';
 import PropTypes from 'prop-types';
 import GalioTheme, { withGalio } from './theme';
 
-class Block extends Component {
-  render() {
-    const {
-      row,
-      flex,
-      center,
-      middle,
-      top,
-      bottom,
-      right,
-      left,
-      shadow,
-      space,
-      fluid,
-      height,
-      shadowColor,
-      card,
-      width,
-      safe,
-      children,
-      style,
-      styles,
-      ...props
-    } = this.props;
+function Block(props) {
+  const {
```

Comment:
you should try destructuring the props in the `props` argument of the function. Like in `Text.js`

Reply:
I have actually destructed the props! 
like this ( Pattern A )
```
const {
    row,
    flex,
    center,
    middle,
    top,
    bottom,
    right,
    left,
    shadow,
    space,
    fluid,
    height,
    shadowColor,
    card,
    width,
    safe,
    children,
    style,
    styles,
    ...rest
  } = props;
```

but not like this,   ( Pattern B )
```
function Block({
    row,
    flex,
    center,
    middle,
    top,
    bottom,
    right,
    left,
    shadow,
    space,
    fluid,
    height,
    shadowColor,
    card,
    width,
    safe,
    children,
    style,
    styles,
    ...rest
  })
```

Reply:
I have used both pattern A and pattern B in the code !
If you could tell the particulat pattern you want the PR to have for the code consistency

Reply:
I feel like the pattern B could be better but we could leave it like that for the moment.